main2.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask, render_template, request, redirect, url_for, session
+
+app = Flask(__name__)
+
+
+app.secret_key = "rrgtqeyq43t43t45t4qter"
+
+@app.get('/submit')
+def submit_get():
+    context = {
+        'login': 'Авторизация'
+    }
+    return render_template('form.html', **context)
+
+
+@app.post('/submit')
+def submit_post():
+    session['login'] = request.form.get('login')
+    session['email'] = request.form.get('email')
+    return redirect(url_for('success'))
```

Comment:
Эти функции можно объединить
```suggestion
@app.route('/submit', methods=['GET', 'POST'])
def submit():
    if request.method == 'POST':
        session['login'] = request.form.get('login')
        session['email'] = request.form.get('email')
        return redirect(url_for('success'))

    context = {
        'login': 'Авторизация'
    }
    return render_template('form.html', **context)
```